sky/utils/duration_utils.py

```diff
@@ -0,0 +1,73 @@
+"""Utils for duration."""
+import time
+
+
+def human_duration_since(unix_timestamp: int) -> str:
```

Comment:
Can this be placed next to readable_time_duration in log_utils instead of a new file?

Reply:
Updated